src/NServiceBus.Core/Recoverability/SecondLevelRetries/CustomSecondLevelRetryPolicy.cs

```diff
@@ -14,7 +14,7 @@ public override bool TryGetDelay(IncomingMessage message, Exception ex, int curr
         {
             delay = customRetryPolicy(message);
 
-            return delay != TimeSpan.Zero;
+            return delay != TimeSpan.MinValue;
```

Comment:
Why this change? This is a behavior breaking change for custom policies. was this intentional?